visualization.shadertoy/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
glew depends for ios are missing in shadertoy
@notspiff is it supposed to work there?